README.md

```diff
@@ -21,11 +21,11 @@ There are some situations where you might want to get specific URL parts:
 
 Either install directly:
 
-`pip install -e git+git@github.com:RRMoelker/django-full-url.git#egg=full_url`
+`pip install -e git+https://github.com/RRMoelker/django-full-url.git#egg=django-full-url`
```

Comment:
Agreed with the https part. But Django pip package default seems to use the egg without the "django-" prefix. Or am I mistaking?